pkg/localkube/proxy.go

```diff
@@ -62,7 +69,11 @@ func StartProxyServer(lk LocalkubeServer) func() error {
 
 	return func() error {
 		// Creating this config requires the API Server to be up, so do it in the start function itself.
-		server, err := kubeproxy.NewProxyServer(config, false, runtime.NewScheme(), "")
+		url := ""
+		if lk.APIServerInsecurePort != 0 {
+			url = lk.GetAPIServerInsecureURL()
```

Comment:
Can this function just return "" if lk.APIServerInsecurePort == 0?

Reply:
Not sure what you mean.  If insecureport is enabled, use that url as the parameter as was done previously.  If not, do what is done currently and use "".
https://github.com/kubernetes/minikube/blob/master/pkg/localkube/proxy.go#L65

Are you suggesting to flip the case so that the url defaults to the insecureURL and then reverse the if statement so that it would set it oppositely?  I don't understand what is gained.

Reply:
I meant 
```golang
func (lk LocalkubeServer) GetAPIServerInsecureURL() {
  if lk.APIServerInsecurePort != 0 {
    	return fmt.Sprintf("http://%s:%d", lk.APIServerInsecureAddress.String(), lk.APIServerInsecurePort)
  }
  return ""
}
```

Reply:
Otherwise, you might end up calling GetAPIServerInsecureURL() with lk.APIServerInsecurePort == 0 and get 

`http://:0`

It might as well give the default value of the return type instead of garbage, that way you can use it how you want to here.

Reply:
done